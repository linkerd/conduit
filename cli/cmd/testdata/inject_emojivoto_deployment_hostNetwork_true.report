
hostNetwork: pods do not use host networking...............................[warn] -- "hostNetwork: true" detected in deployment/web
sidecar: pods do not have a proxy or initContainer already injected........[ok]
supported: at least one resource injected..................................[warn] -- no supported objects found
udp: pod specs do not include UDP ports....................................[ok]

Summary: 0 of 1 YAML document(s) injected

