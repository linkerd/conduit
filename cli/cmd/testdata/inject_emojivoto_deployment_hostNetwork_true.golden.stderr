Error transforming resources: failed to inject deployment/web - pods use host networking
